tidb-lightning/tidb-lightning-backends.md

```diff
@@ -0,0 +1,465 @@
+---
+title: TiDB Lightning Import Modes
+summary: Learn the different import modes of TiDB Lightning.
+aliases: ['/docs/dev/tidb-lightning/tidb-lightning-tidb-backend/','/docs/dev/reference/tools/tidb-lightning/tidb-backend/','/tidb/dev/tidb-lightning-tidb-backend']
+---
+
+# TiDB Lightning Import Modes
+
+TiDB Lightning supports three backends: Importer-backend (default), Local-backend, and TiDB-backend. It determines how `tidb-lightning` delivers data into the target cluster.
+
```

Comment:
How about putting the following three items to a list?

Reply:
Done